CONTRIBUTING.rst

```diff
@@ -223,7 +223,14 @@ syntax above so that the issue(s) will be closed upon merging the PR.
 PR Labels
 ~~~~~~~~~
 
-Add the labels that best describe your PR ('enhancement', 'documentation', etc...
+You **must** add labels to your PRs, as these are used to automatically generate Changelog:
+
+- **Category:** Choose **one** label that describes the `category <https://github.com/shimming-toolbox/shimming-toolbox-py/wiki/Label-definition#Category>`_ (purple labels).
```

Comment:
you can delete the wiki page that you've created to replace with this link (with every new wiki page, we need to maintain the documentation)
```suggestion
- **Category:** Choose **one** label that describes the `category <https://github.com/shimming-toolbox/shimming-toolbox-py/labels?q=bug+documentation+feature+enhancement+testing+installation>`_.
```